storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -820,8 +820,7 @@ struct my_option xb_client_options[] =
    &xtrabackup_encrypt_algo, &xtrabackup_encrypt_algo,
    &xtrabackup_encrypt_algo_typelib, GET_ENUM, REQUIRED_ARG, 0, 0, 0, 0, 0, 0},
 
-  {"encrypt-key", OPT_XTRA_ENCRYPT_KEY, "Encryption key to use.",
-   (G_PTR*) &xtrabackup_encrypt_key, (G_PTR*) &xtrabackup_encrypt_key, 0,
+  {"encrypt-key", OPT_XTRA_ENCRYPT_KEY, "Encryption key to use.",0,0, 0,
```

Comment:
Why these xtrabackup_encrypt_key parameters are removed?

Reply:
rather processing here. it will be processed with help hide_option.
added new case